commit 41212af64ce0f51e02b4fa2a875599cc4d2e9bca
Author: Annaliese McDermond <nh6z@nh6z.net>
Date:   Thu Mar 21 17:58:54 2019 -0700

    ASoC: tlv320aic32x4: Allow 192000 Sample Rate
    
    commit 6d56ee1550b8a81bc63c80051ff78d8d704b09ba upstream.
    
    The clocking and processing blocks are now properly set up to
    support 192000 sample rates.  Allow drivers to ask for that.
    
    Signed-off-by: Annaliese McDermond <nh6z@nh6z.net>
    Signed-off-by: Mark Brown <broonie@kernel.org>

diff --git a/sound/soc/codecs/tlv320aic32x4.c b/sound/soc/codecs/tlv320aic32x4.c
index fea51cc5cad7..3648e251875f 100644
--- a/sound/soc/codecs/tlv320aic32x4.c
+++ b/sound/soc/codecs/tlv320aic32x4.c
@@ -857,7 +857,7 @@ static int aic32x4_set_bias_level(struct snd_soc_component *component,
 	return 0;
 }
 
-#define AIC32X4_RATES	SNDRV_PCM_RATE_8000_96000
+#define AIC32X4_RATES	SNDRV_PCM_RATE_8000_192000
 #define AIC32X4_FORMATS (SNDRV_PCM_FMTBIT_S16_LE | SNDRV_PCM_FMTBIT_S20_3LE \
 			 | SNDRV_PCM_FMTBIT_S24_3LE | SNDRV_PCM_FMTBIT_S32_LE)
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1413_41212af64ce0f51e02b4fa2a875599cc4d2e9bca.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1413_41212af64ce0f51e02b4fa2a875599cc4d2e9bca.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
